src/Illuminate/Foundation/Console/RouteListCommand.php

```diff
@@ -250,8 +250,8 @@ protected function isVendorRoute(Route $route)
     protected function isFrameworkController(Route $route)
     {
         return in_array($route->getControllerClass(), [
-            \Illuminate\Routing\RedirectController::class,
-            \Illuminate\Routing\ViewController::class,
+            '\Illuminate\Routing\RedirectController',
+            '\Illuminate\Routing\ViewController',
```

Comment:
```suggestion
            '\' . Illuminate\Routing\RedirectController::class,
            '\' . Illuminate\Routing\ViewController::class,
```
probably isn't much better, is it?